docs/docs-components/PageHeader.js

```diff
@@ -18,11 +18,19 @@ const buildSourceLinkUrl = (componentName) =>
 
 type Props = {|
   badge?: 'pilot' | 'deprecated',
-  // DEPRECATED: Use `children` instead of `defaultCode`
+  /**
+   * @deprecated : Use `children` instead of `defaultCode`
```

Comment:
Since flow out of the box doesn't yet support deprecated types, we use JSDoc comments instead:
* https://github.com/facebook/flow/issues/4646
* https://jsdoc.app/tags-deprecated.html

Note that in TypeScript, the JSDoc format is also recommended: https://github.com/microsoft/TypeScript/issues/390

Reply:
Neat! 😍